README.md

```diff
@@ -179,7 +179,7 @@ Create the file `/etc/yum.repos.d/libmongocrypt.repo` with contents:
 ```
 [libmongocrypt]
 name=libmongocrypt repository
-baseurl=https://libmongocrypt.s3.amazonaws.com/yum/redhat/$releasever/libmongocrypt/1.5/x86_64
+baseurl=https://libmongocrypt.s3.amazonaws.com/yum/redhat/$releasever/libmongocrypt/1.6/x86_64
```

Comment:
Why refer to 1.6? The most latest release is 1.5.0.

Is using 1.6 required to install the "Unstable Development Distribution Packages"? If so, I suggest noting to replace "1.5" with "1.6" in the "Unstable Development Distribution Packages" section.

Reply:
Oops.  You are absolutely right.  I got a touch carried away.  I'll submit a different PR after I merge #409